packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -244,10 +233,15 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
 
   private _onRenderNavigation = (props: IPanelProps): JSX.Element | null => {
     const { closeButtonAriaLabel, hasCloseButton } = props;
+    const theme = getTheme();
     if (hasCloseButton) {
+      // const iconButtonStyles = this._classNames.subComponentStyles
```

Comment:
Sorry I'll add this to the documentation.  I wrote the code to find out it can't be used until Button is converted.  Seems redundant to delete when this should be the code that is added at that point.  I was going to open an issue to keep this todo on the radar

Reply:
Could always stash the uncommented code in a diff that we can re-apply later once Button has been refactored if we want to keep things clean.

Reply:
There is also an issue to track this for later: #5689 You could refer to the issue in each of tthe commented out code blocks.